query-engine/connector-test-kit-rs/query-tests-setup/src/schema_gen/references.rs

```diff
@@ -134,6 +134,9 @@ pub fn simple_child_references<'a>(
 ) -> Vec<RelationReference<'a>> {
     match *parent_id {
         _ if on_child.is_list() && !on_parent.is_list() => vec![(RelationReference::NoRef)],
+        Identifier::Simple if on_child.is_to_one_opt() && on_parent.is_to_one_opt() => {
+            vec![RelationReference::SimpleParentId(on_child), RelationReference::NoRef]
+        }
```

Comment:
Makes sure we generate optional 1-1 relation inlined on both sides (child _and_ parent)